src/ops/conv_ops/conv2d_transpose.rs

```diff
@@ -481,11 +481,11 @@ fn test_deconv() {
     )
     .unwrap();
 
-    crate::with::<f32, _>(|s| {
+    let out_val = crate::with(|s: &mut crate::Graph<f32>| {
```

Comment:
`crate::with::<f32, _>` no longer compiles (this PR introduces a slight breaking change), so this either needed to be changed to `crate::with::<f32, _, _>`, or changed to take an explicit argument (and I chose the latter).